heat-production-optimization/SavingToCSV.cs

```diff
@@ -0,0 +1,42 @@
+using heat_production_optimization;
+using CsvHelper;
+using System.Globalization;
+
+
+
+namespace heat_production_optimization
+{
+    public class HourlyOptimization
+    {
+        public DateTime timeFrom { get; set; }
+        public DateTime timeTo { get; set; }
+        public static List<HourlyOptimization> HourlyOptimizations{ get; set; } = new List<HourlyOptimization> ();
+        public double GasBolier { get; set; }
+        public double OilBoiler { get; set; }
+        public double GasMotor { get; set; }
+        public double ElectricBoiler { get; set; }
```

Comment:
This can be easily changed to a Dictionary<IUnit, double> which would eliminate the need for a switch case in the optimizer file and would make the saving to csv work with custom boilers as well